fe/fe-core/src/main/java/com/starrocks/transaction/TransactionState.java

```diff
@@ -117,6 +118,11 @@ public static TxnStatusChangeReason fromString(String reasonString) {
                 return null;
             }
 
+            // Pause on unrecoverable error.
+            if (reasonString.contains(TStatusCode.UNRECOVERABLE_ERROR.toString())) {
```

Comment:
Must to convert to string to be compared?

Reply:
Otherwise, the signature of `abortTransaction` would be changed, which may brings many modifications.